### PR TITLE
formats.js

### PR DESCRIPTION
**Pull request by HiTechFlufi**
My messages for my upcoming SSB Pokemon that I bought.

- flufi

### CODE REVIEW

File: config/formats.js [hunk 1]
```diff
@@ -1474,6 +1474,13 @@ exports.Formats = [
 					spd: 1,
 				});
 			}
+			if (name === 'flufi' && !pokemon.illusion) {
+				this.add('-start', pokemon, 'typechange', 'Normal/Ghost');
+				pokemon.types = ["Normal", "Ghost"];
+				this.boost({
+					spe: 4,
```
Comment (ghost) [inline config/formats.js:1481]: > max speed

Comment (ghost) [reply config/formats.js:1481]: Also >normal/ghost

Comment (ghost) [reply config/formats.js:1481]: Also >Travis errors

Comment (DeathlyPlays) [reply config/formats.js:1481]: 6 is max speed

Comment (DeathlyPlays) [reply config/formats.js:1481]: Normal Ghost weakness is Dark OwO what Pokemon is this tho

**Review by ghost — CHANGES_REQUESTED** [review]
Until we know WHAT pokemon we have to work with I'm not gonna merge it

**Pull request closed without merging**